src/components/WorkspaceList/WorkspaceListConfig.tsx

```diff
@@ -60,15 +60,17 @@ export const workspaceFilters = [
 ];
 
 // Actions on each row
-export const workspaceActions: IAction[] = [
+export const workspaceActions = ({ deleteWorkspace }: { deleteWorkspace: (workspaceName: string) => void }): IAction[] => [
```

Comment:
1. In the future we'll need the workspace name from the rowData for Edit workspace as well. Would it be simpler to move `workspaceActions` into the `WorkspaceList` component like this? To avoid having to pass along a reference to `setWorkspaceToDelete`.
<pre><code>
  // Actions on each row
  const workspaceActions: IAction[] = [
    {
      title: 'Edit workspace',
      onClick: () => null,
    },
    {
      title: 'Delete workspace',
      onClick: (event, rowIndex, rowData) => {
        <strong>setSelectedWorkspace</strong>(rowData.name);
      },
    },
  ];

</code></pre>
2. Can we rename `setWorkspaceToDelete` to something more generic like `setSelectedWorkspace` so that we can utilize it for edit workspace and workspace details as well?

Reply:
I think that is an excellent idea.  I went back and forth about moving the workspaceActions when writing the code.  I'll have the change on the next commit

Reply:
@vidyanambiar   Done and ready for re-review